nuxeo-jsf/nuxeo-platform-actions-jsf/src/main/java/org/nuxeo/ecm/platform/actions/jsf/JSFActionContext.java

```diff
@@ -58,15 +59,38 @@ public JSFActionContext(ELContext originalContext, ExpressionFactory expressionF
     @SuppressWarnings("unchecked")
     @Override
     public <T> T evalExpression(String expression, Class<T> expectedType) throws ELException {
+        ValueExpressionDetails veDetails = getValueExpressionDetails(expression, Boolean.class);
```

Comment:
Instead of `Boolean.class` it should be `expectedType`, right?

Reply:
thanks for the review :)

to keep the backward compatibility and avoid any breaking changes I keep it as it was (initially the passed parameter is not used).

have a look [here](https://github.com/nuxeo/nuxeo/blob/10.10/nuxeo-jsf/nuxeo-platform-actions-jsf/src/main/java/org/nuxeo/ecm/platform/actions/jsf/JSFActionContext.java#L94)

Reply:
Fair enough 🙂 
Thanks!